code/modules/mob/inventory.dm

```diff
@@ -155,6 +155,14 @@
 		return FALSE
 	return !held_items[hand_index]
 
+// Can put in any hand?
+/mob/proc/can_put_in_hands(obj/item/I)

```

Comment:
This proc only checks a single pair of active hands, not all of 'em.

Should iterate through `held_items` and check whether any of the indexes have a `null` associated with them.

```
for(var/H in held_items)
    if(!held_items[H])
        return TRUE
return FALSE
```

Reply:
As discussed in Discord, changing it to try and allow more than two hands breaks things terribly. I say we leave it as-is and maybe refactor it if we have a use case for more than two hands.